tests/dem/insertion_plane.cc

```diff
@@ -0,0 +1,131 @@
+/* ---------------------------------------------------------------------
+*
+* Copyright (C) 2019 - 2019 by the Lethe authors
+*
+* This file is part of the Lethe library
+*
+* The Lethe library is free software; you can use it, redistribute
+* it, and/or modify it under the terms of the GNU Lesser General
+* Public License as published by the Free Software Foundation; either
+* version 2.1 of the License, or (at your option) any later version.
+* The full text of the license can be found in the file LICENSE at
+* the top level of the Lethe distribution.
+*
+* ---------------------------------------------------------------------
+
+*
+ * Author: Olivier Gaboriault, Polytechnique Montreal, 2019-
+                           */
+
+/**
+ * @brief Insertion particles using plane insertion class.
```

Comment:
```suggestion
 * @brief Insertion of particles using the plane insertion class.
```

Requires a bit more of a description